src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+

```

Comment:
[Please don't add redundant empty lines to your code.](https://github.com/mate-academy/jv-oop/blob/master/checklist.md#please-dont-add-redundant-empty-lines-to-your-code)
```suggestion
```